src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OdsController.cs

```diff
@@ -12,31 +15,46 @@ namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
     [Produces("application/json")]
     public sealed class OdsController : Controller
     {
+        private readonly IOdsRepository _odsRepository;
+
+        public OdsController(IOdsRepository odsRepository)
+        {
+            _odsRepository = odsRepository;
+        }
+
         [HttpGet]
         [Route("{odsCode}")]
-        public ActionResult GetByOdsCodeAsync(string odsCode)
+        public async Task<ActionResult> GetByOdsCodeAsync(string odsCode)
         {
             if (odsCode is null)
                 throw new ArgumentNullException(nameof(odsCode));
 
             if (string.IsNullOrWhiteSpace(odsCode))
                 return NotFound();
```

Comment:
![image](https://user-images.githubusercontent.com/37105695/79310741-5451fa00-7ef4-11ea-87d8-9a48cceb438f.png)

As per the image above, ods is fine with 'code' being empty. I have to admit I have no idea how ods handles other whitespace characters, however I think we shouldn't be throwing. 

Why shouldn't we be throwing?
I think because of the user experience. Giving a 500 back when they sent a 'space' character seems worse for the UX than returning a 404.